backend/apps/readings/analysis.py

```diff
@@ -3,9 +3,67 @@
 Analysis.py - analyses for dhmit/rereading wired into the webapp
 
 """
+from statistics import stdev
+from pathlib import Path
+
 from .models import StudentResponse
 
 
+def get_sentiments() -> dict:
+    """
+    Returns a dictionary of sentiment scores, with the keys being the word and the values being
+    their score
+
+    :return: dict mapping words to their sentiment scores
+    """
+    sentiment_path = Path('..', 'analysis', 'data', 'sentiments.txt')
+
+    sentiments = dict()
+    with open(sentiment_path, 'r') as file:
+        word = file.readline()
+
+        # We want to handle each word individually, rather than as a whole set
+        while word:
+
+            # This particular file starts lines with '#' for non-sentiment comments, so skip them
```

Comment:
Could all of this from l. 28 - l. 58 be called out to a separate "get_score_from_word(word, sentiments)" function?  Then it can be encapsulated and tested more easily.  Or we wouldn't even need to pass in sentiments if the "word already in dictionary" part was done outside the script.  Then the if statement would also be simplified.